crosswalks/Citation_File_Format_1.2.0.README.md

```diff
@@ -0,0 +1,183 @@
+- CFF `references`: "Reference(s) to other creative works."
+
+#### CodeMeta to CFF
+
+The scope of `references` could be documented more clearly in CFF (https://github.com/citation-file-format/citation-file-format/issues/327).
+CFF's `references` is meant to work like a references list for a paper, 
+e.g., should include prior work that the work builds on.
+Required dependencies are interpreted to clearly constitute prior work that software builds on.
+Therefore, everything that is in `softwareRequirements` should go into `references`.
+
+Optional dependencies and dependencies needed for code development could be assumed 
+to build on the software (i.e., require prior existence of the software).
+Therefore, the software would constitute a `reference` *from* the `softwareSuggestion`,
+but not the other way round.
+I.e., `softwareSuggestions` should *not* be included in `references`.
+
+#### CFF to CodeMeta
+
+Not all CFF `references` can be assumed to be `softwareRequirements`,
+as `references` can contain arbitrary other work that the software builds on.
+The same is even more clearly true for `softwareSuggestions`.
+`references` therefore should not be included automatically in either, 
+without further manual verification.
+
+### `author` <=> `authors`
+
+CFF `authors` is an array of CFF `person` or CFF `entity` objects.
+
+### `citation` <=> `preferred-citation`/`references`/CFF root document
+
+- CodeMeta `citation`: "A citation or reference to another creative work, such as another publication, web page, scholarly article, etc."
+- CFF `preferred-citation`: "A reference to another work that should be cited instead of the software or dataset itself."
+- CFF `references`: "Reference(s) to other creative works."
+- CFF root document: The citation metadata for the software (or dataset) that the `CITATION.cff` file pertains to
+
+#### CodeMeta to CFF
+
+Understanding `citation` in CodeMeta to be the citation for the described software, the relationships are as follows:
+
+- When the `citation` is a citation of the software itself, it maps to the citation metadata provided in the root of a `CITATION.cff` file.
+- When the `citation` is a citation to another work describing the software (a "meta" work, e.g., a software paper in JOSS, a traditional paper about the software), it maps to CFF `preferred-citation`.
+- Diverging from the understanding of `citation` being the citation to the software, when the `citation` is to an unrelated work, it maps to an entry in CFF `references`
+
+#### CFF to CodeMeta
+
+The citation metadata in the root of a `CITATION.cff` file maps to CodeMeta `citation`.
+
+### `license` <=> `license`/`license-url`
+
+#### CodeMeta to CFF
+
+If the CodeMeta `license` contains an SPDX identifier (or its URL) that is recognized in CFF to be a valid value of CFF `license` (list [here](https://github.com/citation-file-format/citation-file-format/blob/main/schema.json#L516-L978)),
```

Comment:
instead of main use 1.2.0

Reply:
Fixed in 8e64a30711c4ce6360360d247b33e479baa50075.